client/src/components/PreviewIcon.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styles from './PreviewIcon.module.css'
+
+export const PreviewIcon = ({
+    onClick,
+    previewStyle,
+    title,
+    body
+}) => {
+
+    const generatePreview = (body) => {
+        return (
+            (body.length) > 20 ? body.split(' ').slice(0, 20).join(" ") : body
+        )
```

Comment:
## Change:
I would add in a constant up the top with the length in it so that people understand why the 20 is there - plus doing "body.split(' ').slice(0,20).join(" ")" thus displays the first 20 words rather then the first 20 characters (if that is what you were going for). I would instead do "body.split('').slice(0,20).join("")" and append ". . ." at the end.

Reply:
Sounds good. Will commit the changes later.

Reply:
@MaxCunningham19
I have addressed this issue in PreviewIcon.module.css.
Other than that, I renamed the function to reflect what it's doing: trimming all the extra characters that the div does not need to contain. 
The constant PREVIEW_CHAR_LIMIT simply serve as an overestimate (I guess if we decide to make these previews resizable, in the future I could try to compute it based on the dimensions of div).
Anyway, the more precise adjustment and append of ellipsis will be then handled at render by CSS.